### PR TITLE
🐛AMP STORY : Fixed localization for simplified Chinese

### PR DESCRIPTION
**Pull request by jenkey2011**
Localization for simplified Chinese is not accurate or even wrong~

### CODE REVIEW

**Comment by jenkey2011** [comment]
I signed it!

File: extensions/amp-story/1.0/_locales/zh.js [hunk 1]
```diff
@@ -43,19 +43,19 @@ export default {
...
   [LocalizedStringId.AMP_STORY_SHARING_CLIPBOARD_SUCCESS_TEXT]: {
-    string: '链接成功！',
+    string: '链接复制成功！',
   },
   [LocalizedStringId.AMP_STORY_SHARING_PROVIDER_NAME_EMAIL]: {
-    string: '电邮',
+    string: '邮件',
   },
   [LocalizedStringId.AMP_STORY_SHARING_PROVIDER_NAME_FACEBOOK]: {
-    string: '脸书',
```
Comment (kevinkassimo) [inline extensions/amp-story/1.0/_locales/zh.js]: I believe that '脸书' should probably also be an name understandable in mainland China.

Comment (jenkey2011) [reply extensions/amp-story/1.0/_locales/zh.js]: Hi @kevinkassimo , thanks for your review!

Glad to know that you speak Chinese :-)

And yes, `'脸书'` do also be an name understandable in mainland China. But due to some irresistible causes you know……  Most people in mainland China can't visit Facebook. They only heard about it. As far as I know, most people ( at least people around me ), would call 'Facebook' directly, not `'脸书'` or `'脸谱'`.

In a word, Facebook in mainland China, is not 'transliteration', nor 'literal translation', but 'not translation' mostly.

The places where people can normally visit Facebook, I am not sure, will not be discussed here.

Comment (kevinkassimo) [reply extensions/amp-story/1.0/_locales/zh.js]: Good point

**Comment by mrjoro** [comment]
Thanks for the translations @jenkey2011, and sorry for the delay in getting this reviewed!

**Comment by gmajoulet** [comment]
Thank you!

**Pull request merged**